features/integration/compile.feature

```diff
@@ -27,5 +27,5 @@ Feature: Compile
     When I compile a teal program <teal> with mapping enabled
     Then the resulting source map is the same as the json <sourcemap>
     Examples:
-      | teal                  | sourcemap                                    |
-      | "programs/quine.teal" | "v2algodclient_responsejsons/sourcemap.json" |
\ No newline at end of file
+      | teal                  | sourcemap                                     |
+      | "programs/quine.teal" | "v2algodclient_responsejsons/sourcemap2.json" |
```

Comment:
@tzaffi For my understanding:  Can you share what motivated the `sourcemap2.json` introduction?

Reply:
Formerly `sourcemap.json` was used by both a unit test (look for the tag `@unit.sourcemap`) as well as an integration test (tag `@compile.sourcemap`). Somehow, the expectations of each of these tests contradict each other (at least in the Python SDK). When I modified the integration test to use the new fixture, I was able to get it the tests to pass, but I haven't looked closely at what is the exact problem. I was planning to reach out to @barnjamin as I believe this is related to #353.

Reply:
@tzaffi Got it - thanks for the explanation.

Seems a bit discomforting, but not enough to hold the PR.  Resolving.